.github/workflows/codespaces-test.yml

```
@@ -53,27 +29,24 @@
 
     - name: Run tests
       env:
-        POSTGRES_HOST: localhost
-        REDIS_HOST: localhost
+        CODESPACES: true
         DJANGO_SETTINGS_MODULE: config.settings
         PYTHONPATH: .
       run: |
         # Run tests with better error handling
         pytest -v --tb=short || true
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)